### PR TITLE
fix ctf design issues and pictures

### LINKED ISSUE #59: Bad picture position -CTF
image

### LINKED ISSUE #61: Object overrides text
https://bsidestlv.com/CTF/2020/

image

### PR DESCRIPTION
**Pull request by omercnet**
fixes multiple issues with CTF page 

also fixes #61 and fixes #59

### CODE REVIEW

**Comment by omercnet** [comment]
@barnhartguy lmkwyt and I'll update previous years too

File: content/_index.md [hunk 1]
```diff
@@ -30,14 +30,16 @@ We are excited to announce that BSidesTLV 2020 will take place in hybrid format!
       icon="cfp" >}}
 
     {{< button-link
-      url="//tickets.bsidestlv.com/bsidestlv/2020"
+      url="//tickets.bsidestlv.com/bsidestlv/2020/5/"
```
Comment (barnhartguy) [inline content/_index.md:33]: the register button on the main page should lead to the registration page which has the sponsor logos

**Comment by barnhartguy** [comment]
1. The CTF banner should be larger than HTB and on the left hand side
2. The buttons should be below the rules (i actually want them side by side, as they are easily missed or overshaddow the rules)
3. on mobile the banners become very small - and still side by side, it would be better to have them larger and on top of each other
4. I suggest updating the timer code to remove the timer instead of "boom"

**Comment by barnhartguy** [comment]
looks ok to me

File: content/_index.md (hunk 1 above)
Comment (omercnet) [reply content/_index.md:33]: not related to this issue

**Pull request merged**